zetaclient/evm_client.go

```diff
@@ -57,7 +57,12 @@ type EVMLog struct {
 }
 
 const (
-	DonationMessage = "I am rich!"
+	DonationMessage    = "I am rich!"
+	TopicsZetaSent     = 3 // https://github.com/zeta-chain/protocol-contracts/blob/d65814debf17648a6c67d757ba03646415842790/contracts/evm/ZetaConnector.base.sol#L34
+	TopicsZetaReceived = 4 // https://github.com/zeta-chain/protocol-contracts/blob/d65814debf17648a6c67d757ba03646415842790/contracts/evm/ZetaConnector.base.sol#L45
+	TopicsZetaReverted = 3 // https://github.com/zeta-chain/protocol-contracts/blob/d65814debf17648a6c67d757ba03646415842790/contracts/evm/ZetaConnector.base.sol#L54
+	TopicsWithdrawn    = 3 // https://github.com/zeta-chain/protocol-contracts/blob/d65814debf17648a6c67d757ba03646415842790/contracts/evm/ERC20Custody.sol#L43
+	TopicsDeposited    = 2 // https://github.com/zeta-chain/protocol-contracts/blob/d65814debf17648a6c67d757ba03646415842790/contracts/evm/ERC20Custody.sol#L42C1-L42C1
```

Comment:
Can we add a comment explaining what these represent?

Reply:
sure

Reply:
[commit](https://github.com/zeta-chain/node/pull/1539/commits/cb257f672246dee79554c565f711dec5b3ab1345)